docs/src/main/sphinx/connector/postgresql.rst

```diff
@@ -139,6 +139,8 @@ statements, the connector supports the following features:
 
```

Comment:
@hashhar does this connector support COMMENT? Might be out of scope for this PR but IIRC it's not available in any JDBC connectors..?

Reply:
Some JDBC connectors support COMMENT. e.g. Postgres and Oracle supports COMMENT ON COLUMN (but not on tables). Rest of them do not.

Reply:
Got it, thanks.